test/suites/alpha/integration/extended_resources_test.go

```diff
@@ -488,3 +606,181 @@ func ExpectHabanaDevicePluginCreated() {
 		},
 	})
 }
+
+func ExpectNeuronDevicePluginCreated() {
```

Comment:
Do we know if this is shipped with the AMIs or is this always required to be installed?